client/app/bundles/course/lesson-plan/reducers/utils.js

```diff
@@ -53,8 +53,14 @@ function sortByStartAt(a, b) {
  * @return {Array.<{ milestone: Object, items: Array }>}
  */
 export function groupItemsUnderMilestones(items, milestones) {
-  const sortedItems = [...items].sort(sortByStartAt);
   const sortedMilestones = [...milestones].sort(sortByStartAt);
+  const sortedItems = [...items].sort((a, b) => {
```

Comment:
Add a comment to the function description explaining the sort order.

Reply:
Done.